docs/resources/annotation.md

```diff
@@ -0,0 +1,97 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+subcategory: "Generic"
+layout: "aci"
+page_title: "ACI: aci_annotation"
+sidebar_current: "docs-aci-resource-aci_annotation"
+description: |-
+  Manages ACI Annotation
+---
+
+# aci_annotation #
+
+Manages ACI Annotation
+
+## API Information ##
+
+* Class: [tagAnnotation](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/tagAnnotation/overview)
+
+* Supported in ACI versions: 3.2(1l) and later.
+
+* Distinguished Name Formats
+  - `Too many DN formats to display, see model documentation for all possible parents.`
```

Comment:
Add a link to the tagAnnotation object page in devnet model doc in this sentence.